### ISSUE TITLE
뷰 생성 및 결합

### ISSUE THREAD

**Issue by goccd** [proghjy/SwiftUI-Tutorial#1]
MapKit, Image, Text 등의 구성요소를 Stack을 사용해 구성 및 결합하여 랜드 마크의 세부 사항을 표시하는 화면을 구현합니다

**Comment by goccd** [comment]
```swift
Image("turtlerock")
        .clipShape(Circle())
        .overlay(
            Circle().stroke(Color.white, lineWidth: 4))
        .shadow(radius: 10)
```

이미지를 원으로 자르고, 테두리 라인 및 그림자 효과를 적용

**Comment by goccd** [comment]
Q1. When creating a custom SwiftUI view, where do you declare the view’s layout?
1. In the view’s initializer.
2. In the body property.
3. In the layoutSubviews() method.

Q2. Which layout renders from the following view code?
```swift
var body: some View {
    HStack {
        CircleImage()
        VStack(alignment: .leading) {
            Text("Turtle Rock")
                .font(.title)
            Text("Joshua Tree National Park")
        }
    }
}
```
1. ![](https://docs-assets.developer.apple.com/published/c8c89e6d7f/90bd5db7-c005-48c0-b135-f294116a5ad0.png)
2. ![](https://docs-assets.developer.apple.com/published/84c871f444/eeb03e46-673d-4cf4-8c1d-8310e412eb6c.png)
3. ![](https://docs-assets.developer.apple.com/published/65c0b8f6e9/1d29d56a-0a1c-4012-9043-0c6f572fb55d.png)

Q3. Which of these is a correct way to return three views from a custom view’s body property?
1.
```swift
VStack {
   Text("Turtle Rock")
      .font(.title)
   Divider()
   Text("Joshua Tree National Park")
}
```
2. 
```swift
[
    Text("Turtle Rock").font(.title),
    Divider(),
    Text("Joshua Tree National Park")
]
```
3.
```swift
Text("Turtle Rock")
    + Divider()
    + Text("Joshua Tree National Park")
```

Q4. Which is the correct way to use modifier methods to configure a view?
1.
```swift
var text = Text("Hello world!")
text.font(.title)
text.foregroundColor(.purple)
return text
```
2.
```swift
var text = Text("Hello world!")
text.font = .title
text.foregroundColor = .purple
return text
```
3.
```swift
Text("Hello world!")
   .font(.title)
   .foregroundColor(.purple)
```
